### PR TITLE
change readme + added some function into config

### PR DESCRIPTION
**Pull request by studplus**
added geofence explanation to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -620,6 +622,22 @@ Predefine sharing all raids to the chats -100111222333 and -100444555666, except
...
+
+You can receive Raids from Scanner like MAD via Webhook.
+For that you need to setup
+`"WEBHOOK_CHATS":"-100444555666"`
+or by Raidlevel `"WEBHOOK_CHATS_LEVEL_5":"-100444555666"`
+All raids will be published in these chats.
+
+#### Filter Raids from Webhook / geoconfig.json
+
+If you got different Chats for different Areas you can setup them in
```
Comment (Artanicus) [inline README.md:635]:
```suggestion
If you have multiple Chats for different Areas you can setup them in
```

File: README.md [hunk 2]
```diff
@@ -620,6 +622,22 @@ Predefine sharing all raids to the chats -100111222333 and -100444555666, except
 `"SHARE_CHATS":"-100111222333,-100444555666"`
 `"SHARE_CHATS_LEVEL_5":"-100444555666"`
 
+#### Raids from Webhook
+
+You can receive Raids from Scanner like MAD via Webhook.
```
Comment (Artanicus) [inline README.md:627]:
```suggestion
You can receive Raids from a mapping system such as MAD via Webhook.
```

File: README.md [hunk 3]
```diff
@@ -620,6 +622,22 @@ Predefine sharing all raids to the chats -100111222333 and -100444555666, except
...
+For that you need to setup
+`"WEBHOOK_CHATS":"-100444555666"`
+or by Raidlevel `"WEBHOOK_CHATS_LEVEL_5":"-100444555666"`
+All raids will be published in these chats.
+
+#### Filter Raids from Webhook / geoconfig.json
+
+If you got different Chats for different Areas you can setup them in
+`"WEBHOOK_CHATS_LEVEL_5_0":"-100444555666"` regarding to your configuration in the geoconfig.json.
```
Comment (Artanicus) [inline README.md:636]:
```suggestion
`"WEBHOOK_CHATS_LEVEL_5_0":"-100444555666"` matching with your configuration in the geoconfig.json.
```

**Comment by Chefkeks** [comment]
Can you do me a favor?
Add a trimming of all the 0‘s / zeros at the end of each maps link. Currently the look like 
https://maps.google.com/?daddr=49.4418720000000000,7.7530210000000000 in our raidbot. But https://maps.google.com/?daddr=49.441872,7.753021 would be much nice qnd consume less chars - as captions (raidpicture feature) only allow 1024, every char less is good :)

**Comment by Chefkeks** [comment]
PS: 👍 Good work

**Comment by studplus** [comment]
@Chefkeks trimming the lon and lat -> done

**Comment by Chefkeks** [comment]
Hmm 🤔 
Cutting at the 6th digit after the decimal point will „break“ some lat / lon. A lot of coords in our database have 8 and some even 9 digits. Therefore my idea was to cut the 0‘s and not after a specific digit.

**Comment by Artanicus** [comment]
The rounding seems like a decent way to do it in my opinion, keeps the length to a known one regardless of arbitrary precision. It's only used on output to Maps and will allow people to find the gym anyway even if it's 30cm off the target :-)

New Maps stuff LGTM

**Comment by Chefkeks** [comment]
I know it won’t be such a huge difference, but my thought is like: It’s in the database with x digits + the 0‘s, so trim the 0‘s to shorten without affecting the actual coordinates. But I agree, 30cm / 1 feet should not make a huge difference

**Pull request merged**